src/maxKey.js

```diff
@@ -9,11 +9,19 @@
  * @returns {number} The key with the largest value
  */
 function maxKey(obj) {
-  // This is your job. :)
-}
+    return Object.keys(obj).filter(x => {
```

Comment:
1. In modern JS, you can use the [spread operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to turn an array into a list of arguments, e.g., `Math.max(...Object.values(obj))`
1. That said, you don't want to do this here. You're calling `Math.max` once for every key in the object. If there are 20 key/value pairs, you're looking at `20 * 20` key/value pairs to find the key with the largest value. Try rewriting it so you only iterate over the keys once.